cpp/src/io/orc/reader_impl.cu

```diff
@@ -641,6 +642,7 @@ void reader::impl::decode_stream_data(cudf::detail::hostdevice_2dvector<gpu::Col
     update_null_mask(chunks, out_buffers, stream, _mr);
   }
 
+  rmm::device_scalar<unsigned long long int> error_count(0, stream);
```

Comment:
`unsigned long long int` seems very long and ambiguous. Please use the fixed width type, such as `uint64_t`.

Reply:
In addition, can we just use `uint32_t` instead? I'm not sure if `uint64_t` is supported in `atomicAdd` below.

Reply:
These are atomicAdd overloads:
```
int atomicAdd(int* address, int val);
unsigned int atomicAdd(unsigned int* address,
 unsigned int val);
unsigned long long int atomicAdd(unsigned long long int* address,
 unsigned long long int val);
 ```
 I can use a smaller type, but should not be using `uint32_t` or `uint64_t`, AFAIK.

Reply:
Went over the uses of atomicAdd in libcudf, and, to my surprise, we often use it with `size_type`. I'll change this to `size_type`.

Reply:
> I can use a smaller type, but should not be using uint32_t or uint64_t, AFAIK.

`unsigned long long int` should just be alias of `uint64_t` if I was not wrong.